src/BotPlutusInterface/TimeSlot.hs

```diff
@@ -108,11 +111,12 @@ posixTimeRangeToContainedSlotRangeIO
     nodeInfo <- liftIO $ mkNodeInfo pabConf
     sysStart <- newET $ querySystemStart nodeInfo
     eraHistory <- newET $ queryEraHistory nodeInfo
+    pparams <-
```

Comment:
this snippet shows in two places. Extracted to mkPParams in `Avoid repeating`.

Reply:
👍